### ISSUE TITLE
Encountered errors when viewing ticket

### ISSUE THREAD

**Issue by imarklee** [freescout-helpdesk/freescout#1725]
Hey guys,

Encountered the following fatal errors when trying to view a thread:

````bash
[2022-02-12 15:11:09] production.ERROR: DateTime::__construct(): Failed to parse time string (Sat, 12 Feb 2022 11:47:03 +0800 (GMT+08:00)) at position 36 (+): Double timezone specification (View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) 
(View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) 
(View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) 
{"userId":1,"email":"xxxx@xxxx","exception":"[object] (ErrorException(code: 0): 
DateTime::__construct(): Failed to parse time string (Sat, 12 Feb 2022 11:47:03 +0800 (GMT+08:00)) at position 36 (+): Double timezone specification (View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) 
(View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) 
(View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) at /home/admin/helpdesk/overrides/nesbot/carbon/src/Carbon/Carbon.php:548, 
ErrorException(code: 0): DateTime::__construct(): Failed to parse time string (Sat, 12 Feb 2022 11:47:03 +0800 (GMT+08:00)) at position 36 (+): 
Double timezone specification (View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) 
(View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) at /home/admin/helpdesk/overrides/nesbot/carbon/src/Carbon/Carbon.php:548, 
ErrorException(code: 0): DateTime::__construct(): Failed to parse time string (Sat, 12 Feb 2022 11:47:03 +0800 (GMT+08:00)) at position 36 (+): 
Double timezone specification (View: /home/admin/helpdesk/resources/views/conversations/partials/thread.blade.php) at /home/admin/helpdesk/overrides/nesbot/carbon/src/Carbon/Carbon.php:548, 
Exception(code: 0): DateTime::__construct(): Failed to parse time string (Sat, 12 Feb 2022 11:47:03 +0800 (GMT+08:00)) at position 36 (+): Double timezone specification at /home/admin/helpdesk/overrides/nesbot/carbon/src/Carbon/Carbon.php:548)
[stacktrace]
````

**Comment by freescout-helpdesk** [comment]
Could you please post here headers of the email. You can find them in the thread dropdown menu under Show Original

**Comment by imarklee** [comment]
Sure, this is the headers:

```
Delivered-To: xxx@xxx
Received: by 2002:a05:6020:5f87:b0:17d:a12f:84aa with SMTP id mf7csp1264006wdb;
        Fri, 11 Feb 2022 19:48:07 -0800 (PST)
X-Google-Smtp-Source: ABdhPJzswvVdNyFnPvjp4fGrCqDeTMhIxrcPYt7k841g4Xb1Jq2+7DtRzN8Lsww1N+Gn4DWaVNbk
X-Received: by 2002:a17:90b:38cc:: with SMTP id nn12mr3523075pjb.228.1644637686777;
        Fri, 11 Feb 2022 19:48:06 -0800 (PST)
ARC-Seal: i=1; a=rsa-sha256; t=1644637686; cv=none;
        d=google.com; s=arc-20160816;
        b=fE9LMDzJeTrkVDyofTSkz3XurIAOmYZjG5LWnv4DpyNV/b3c9M7P0xhuWRIdO7Ibpx
         yeMCAKU1zIrUoMAQMRo8+VYcrfraJVzDmnNUFw1HZvFJeTCWwG5OYgSynSuQVoD1BUwA
         rQP1YAVGfg1NsQ0Sb2tVMjXSxlVCbL/xRikTg3WC2ZF+B8In9zrxWujuZDTmJaUvXhYE
         r5+jUHmfSD7tra9nOqgitW2/4Ran1X2ZsMyjDrltpWCBX2bUlOC4I67pxazmt2Xvbc9B
         hkIfstrBslpYwwwZnuiPQI2Ivk7hF2jCNRP5+Xnd2TRVdDEMW9nb3UIzpyxcyj5FhYkg
         odGg==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=date:from:mime-version:in-reply-to:subject:to:message-id;
        bh=kdaMO1jLIPi9RnBQl7PNa0WcLDxtzmQL5UiakG5S2nM=;
        b=PnOVQaeJAZ64ltx6T7gsMSc6N7AJoE6myOjpoZuwAnUGUdNpi/9t1W99Rji69bsNoE
         M9Q52WkLbOYmx83CwYe7WHbMzF86sKGEVoP2Mm/qFyI4bf00ke9tVjMFAHVN1OBl9jS+
         F/hN1IrNsdVXw2cM9ikHCQ4wYCuGsHNFY8J4NmNenQGxrH9WS8MImvduUJwc5EAmGgnD
         Ss13H8f7MjKYFRprBfgmJZ/ET5EMl4p5cxpLInSHLoVq+GCIPSzCOeWBnUw16uFHGGEt
         +2mAzBtH3YBU2QeT6oS7nVrz4/m2cMPL6mkFluKeEvtTM6k8ttOcOoCKs5/TYtIWduIL
         2NvA==
ARC-Authentication-Results: i=1; mx.google.com;
       spf=pass (google.com: domain of yyy@yyy designates 123.123.123.123 as permitted sender) smtp.mailfrom=yyy@yyy
Return-Path: <yyy@yyy>
Received: from mail-m255227.qiye.163.com (mail-m255227.qiye.163.com. [123.123.123.123])
        by mx.google.com with ESMTP id s11si5635662pjg.9.2022.02.11.19.48.05
        for <xxx@xxx>;
        Fri, 11 Feb 2022 19:48:06 -0800 (PST)
Received-SPF: pass (google.com: domain of yyy@yyy designates 123.123.123.123 as permitted sender) client-ip=123.123.123.123;
Authentication-Results: mx.google.com;
       spf=pass (google.com: domain of yyy@yyy designates 123.123.123.123 as permitted sender) smtp.mailfrom=yyy@yyy
Received: from ibktek.com (localhost [127.0.0.1])
	by mail-m121164.qiye.163.com (HMail) with ESMTP id 17D963A00DD
	for <xxx@xxx>; Sat, 12 Feb 2022 11:47:03 +0800 (CST)
Content-Type: multipart/alternative; BOUNDARY="=_Part_517537_468580332.1644637623089"
Message-ID: <AMcAigAQB300bttMVCMiJqrC.3.1644637623089.Hmail.yyy@yyy>
To: Xxx <xxx@xxx>
Subject: =?UTF-8?B?UmU6WyMyODhdIE5ldyBJbnF1aXJ5IC0gU3RhY2tJZGVhcyBDb250YWN0IEZvcm0gKEVhc3lTb2NpYWwp?=
X-Priority: 3
X-Mailer: HMail Webmail Server V2.0 Copyright (c) 2016-163.com
X-Originating-IP: 222.222.222.222
In-Reply-To: <0100017eebf34cad-f1d4b6ed-73c8-499b-92e3-5f6000216567-000000@email.amazonses.com>
MIME-Version: 1.0
Received: from yyy@yyy( [222.222.222.222) ] by ajax-webmail ( [127.0.0.1] ) ; Sat, 12 Feb 2022 11:47:03 +0800 (GMT+08:00)
From: yyy@yyy
Date: Sat, 12 Feb 2022 11:47:03 +0800 (GMT+08:00)
X-HM-Spam-Status: e1kfGhgUHx5ZQUtXWQgPGg8OCBgUHx5ZQUlOS1dZCBgUCR5ZQVlLVUtZV1
	kWDxoPAgseWUFZKDYvK1lXWShZQU83V1ktWUFJV1kPCRoVCBIfWUFZQksfGFZOHx5LQk9DQx9NHh
	9VAhYTFhoSFyQUDg9ZV1kWGg8SFR0UWUFZVUtZBg++
X-HM-Sender-Digest: e1kJHlYWEh9ZQUhISkhJTkJOTUxIN1dZDB4ZWUEPCQ4eV1kSHx4VD1lB
	WUc6Nhg6Ehw6KjlIS0sZDw82LTg2EjEKCThVSFVKTU9PTUhMTUlISExCVTMWGhIXVQwSFxcSGhY7
	EhkQDx4QVRgUFkVZV1kSC1lBWUpLT1VJT0JVSkxIVUpCS1lXWQgBWUFJSUxNN1dZFAsPEhQVCFlB
	SzcG
X-HM-Tid: 0a7eec0a8fd3b04ekuuu17e67ada025
```

**Comment by freescout-helpdesk** [comment]
Fixed in Sender Time Zone Module v1.0.2
